frontend/src/types/Location.ts

```diff
@@ -1,5 +1,5 @@
 export type Location = {
-    id: string,
+    locationId: string,
```

Comment:
Why do you call this attribute 'locationId' instead of 'id'? You could suggest to undo the rename, because you have just one field 'id' in your Location type